spec/controllers/games_controller_spec.rb

```diff
@@ -7,4 +7,13 @@
       expect(response).to have_http_status(:success)
     end
   end
+
+  describe 'games#create action' do
+    it 'should successfully create a new game in our database' do
+      post :create, params: { game: { user_id_black: 15 } }
+      @game = Game.last
+      expect(response).to redirect_to game_path[:@game.id]
```

Comment:
Well, we're PART of the way there. (Progress!).
Using parentheses changed the error message to:
     Failure/Error: expect(response).to redirect_to game_path(@game.id)
     
     NoMethodError:
       undefined method `id' for nil:NilClass

I tried @game and @game.id and even Game.last.id and still got the same error message.
(In rails console when I ran Game.create and then Game.last.id, it showed the id fine. 
     Does this mean we're not actually creating a game?
     Is there something wrong with the line that is supposed to create a game? I have:
         post :create, params: { game: { user_id_black: 15 } }
I tried messing with it, without success...